sw/device/lib/dif/dif_otbn.c

```diff
@@ -197,11 +197,3 @@ dif_result_t dif_otbn_set_ctrl_software_errs_fatal(const dif_otbn_t *otbn,
 
   return kDifOk;
 }
-
-size_t dif_otbn_get_dmem_size_bytes(const dif_otbn_t *otbn) {
```

Comment:
These DIF apis are meant to be a layer of abstraction between the hardware and any test code.